core/src/main/java/io/confluent/rest/auth/AuthUtil.java

```diff
@@ -128,9 +128,6 @@ private static ConstraintMapping createConstraint(
     final ConstraintMapping mapping = new ConstraintMapping();
     mapping.setConstraint(constraint);
     mapping.setMethod("*");
-    if (authenticate && AuthUtil.isCorsEnabled(restConfig)) {
```

Comment:
@maverick64 did you verify this works with basic auth?

that i can make an OPTIONS call from the browser and the 401 does not prevent the return, or at least prompts the user for a login

Reply:
I didn't even know this code path gets used by C3. Browser could be sending unauthenticated OPTION request. @jeffhuang26 can you comment on this as well?